### PR TITLE
Update SCT3 to account for new LES cases

### PR DESCRIPTION
**Pull request by costachris**
## Changes
Update SCT3 to account for new LES library included in TC `v1.2.0`. Specifies 200 LES cases instead of the full 203 to allow for more batch size options. 

## Checklist before requesting a review / merging
- [x] I have formatted the code using `julia --project=.dev .dev/climaformat.jl .`
- [x] I have updated all test manifests using `julia --project .dev/up_deps.jl .` with Julia 1.7.3.
- [x] If core features are added, I have added appropriate test coverage.
- [x] If new functions and structs are added, they are documented through docstrings.

### CODE REVIEW

File: experiments/SCT3_benchmark/global_parallel/init.sbatch [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #SBATCH --time=2:00:00   # walltime
 #SBATCH --ntasks=1       # number of processor cores (i.e. tasks)
 #SBATCH --nodes=1         # number of nodes
-#SBATCH --mem-per-cpu=20G   # memory per CPU core
+#SBATCH --mem-per-cpu=50G   # memory per CPU core
```
Comment (ilopezgp) [inline experiments/SCT3_benchmark/global_parallel/init.sbatch:6]: Do we need so much memory? This will make us wait longer in the queue

Comment (costachris) [reply experiments/SCT3_benchmark/global_parallel/init.sbatch:6]: Unfortunately this is the lowest memory specification (in increments of 10gb) that worked for me. (ie. 40gb led to an OOM error)

**Review by ilopezgp — APPROVED** [review]
LGTM

**Pull request merged**